qml/Ping360EthernetConfiguration.qml

```diff
@@ -0,0 +1,118 @@
+import QtQuick 2.7
+import QtQuick.Controls 2.2
+import QtQuick.Controls.Material 2.1
+import QtQuick.Layouts 1.3
+import QtQml.Models 2.2
+
+import DeviceManager 1.0
+import Ping360HelperService 1.0
+import StyleManager 1.0
+
+PingPopup {
+    id: root
+
+    closePolicy: Popup.NoAutoClose
+    property var connection: undefined
+
+    // Default size
+    width: 300
+    height: 280
+
+    function configureConnection(connection) {
+        open()
+        root.connection = connection
+    }
+
+    Label {
+        id: title
+        text: "Ping360 Ethetnet configuration"
+        anchors.horizontalCenter: parent.horizontalCenter
+    }
+
+    ListModel {
+        id: ethernetModes
+        ListElement {
+            name: "DHCP Client"
+            info: "Allows to connect as a normal ethernet client to a DHCP server."
+        }
+        ListElement {
+            name: "Static IP"
+            info: "Used to set a fixed IP address in a personalized network environment.<br>⚠ <strong>Ensure to configure your subnet if necessary</strong> ⚠"
+        }
+    }
+
+    GridLayout {
+        id: grid
+        columns: 2
+        anchors.top: title.bottom
+        anchors.right: parent.right
+        anchors.left: parent.left
+        anchors.bottom: parent.bottom
+
+        RowLayout {
+            Layout.columnSpan: 2
+            Layout.fillWidth: true
+            Label {
+                text: "Type:"
+            }
+            PingComboBox {
+                id: comboBox
+                model: ethernetModes
+                textRole: "name"
+                Layout.fillWidth: true
+            }
+        }
+
+        Text {
+            text: ethernetModes.get(comboBox.currentIndex).info
+            color: comboBox.currentIndex == 1 ? "red" : "black"
+            horizontalAlignment: Text.AlignHCenter | Text.AlignJustify
+            wrapMode: Text.Wrap
+            Layout.columnSpan: 2
+            Layout.fillWidth: true
+        }
+
+        PingTextField {
+            id: ipTextField
+            title: "IP address:"
+            text: "192.168.2.4"
```

Comment:
can/should we use the current ip address instead?

Reply:
Maybe @brianhBR can point which is the best static IP address for it.

Reply:
I don't have any specific IP in mind as long as it is on the same network. I like the idea of warning the user if they pick something other than 192.168.2.xxx.